gs-gateway/src/main/java/pl/cyfronet/gsg/security/LayerSecurityGatewayFilter.java

```diff
@@ -44,7 +44,7 @@
         if (accessType == AccessType.OPEN) {
             return chain.filter(exchange);
         } else if (accessType == AccessType.EUMETSAT) {
-            if (isFresh(request)) {
```

Comment:
Wouldn't it better to write
```
return isFreshAndNotOnTheHour(request) ? requiresLicense(exchange, chain, layer) : chain.filter(exchange);

Reply:
I prefer the expanded form, it's more readable by me and easier to follow the logical flow